ballerina/field.bal

```diff
@@ -25,10 +25,12 @@ public class Field {
     private final __Type fieldType;
     private (string|int)[] path;
     private string[] resourcePath;
+    private readonly & Interceptor[] fieldInterceptors;
 
     isolated function init(parser:FieldNode internalNode, __Type fieldType, service object {}? serviceObject = (),
                            (string|int)[] path = [], parser:RootOperationType operationType = parser:OPERATION_QUERY,
-                           string[] resourcePath = [], any|error fieldValue = ()) {
+                           string[] resourcePath = [], any|error fieldValue = (),
+                           readonly & Interceptor[] fieldInterceptors = []) {
```

Comment:
Seems like we are not using the `fieldInterceptors` argument

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1316/commits/7084ba5bbf3595fb9cd9714cf8e00cd03db7b612